PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -27,6 +27,9 @@
         /// Enable masking of images that likely originated from user's photo library
         /// Experimental support (UIKit only)
         /// Default: true
+        ///
+        /// - Note: Deprecated
+        @available(*, deprecated, message: "This property has no effect and will be removed in the next major release. To manually mask user photos, use `ph-no-capture` accessibility label or an appropriate view modifier in your app.")
```

Comment:
can we link to https://posthog.com/docs/session-replay/privacy?tab=iOS instead of "appropriate view modifier in your app"